src/wp-includes/theme-compat/header.php

```diff
@@ -18,44 +18,33 @@
 <!DOCTYPE html>
 <html <?php language_attributes(); ?>>
 <head>
-<link rel="profile" href="https://gmpg.org/xfn/11" />
-<meta http-equiv="Content-Type" content="<?php bloginfo( 'html_type' ); ?>; charset=<?php bloginfo( 'charset' ); ?>" />
-
-<title><?php echo wp_get_document_title(); ?></title>
-
-<link rel="stylesheet" href="<?php bloginfo( 'stylesheet_url' ); ?>" type="text/css" media="screen" />
-<link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
-
-<?php if ( file_exists( get_stylesheet_directory() . '/images/kubrickbgwide.jpg' ) ) { ?>
-<style type="text/css" media="screen">
-
+	<meta charset="<?php bloginfo( 'charset' ); ?>" />
+	<title><?php echo wp_get_document_title(); ?></title>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link rel="profile" href="https://gmpg.org/xfn/11" />
+	<style> :where(#header) { padding-left: 5%; padding-right: 5%; } </style>
```

Comment:
The `style` tag adds padding specifically for the header with block themes, so it should line up with the forms instead of touching the edge. I used `:where()` so a stylesheet could override that easily.